lua/telescope/deprecated.lua

```diff
@@ -0,0 +1,46 @@
+
+local deprecated = {}
```

Comment:
deprecated is currently only checked for options directly passed to a builtin (and then picker) but we could also check if the setup table has one of the deprecated keys. Help people clean that up ;)